org.eclipse.help.ui/src/org/eclipse/help/ui/internal/browser/embedded/EmbeddedBrowser.java

```diff
@@ -304,31 +305,50 @@ public void changing(LocationEvent e) {
 	}
 
 	private void initializeStatusBar(Browser browser) {
+
+		// Text
 		browser.addStatusTextListener(event -> {
-			event.text = event.text.replaceAll("&", "&&"); //$NON-NLS-1$ //$NON-NLS-2$
+			event.text = event.text.replace("&", "&&"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Seems like a bugfix, isn't it worth a dedicated PR?

Reply:
That's just a (performance) clean-up, not a bugfix.

Reply:
I see and I have reverted the clean-up.